lib/platform/node/node_http.js

```diff
@@ -35,13 +35,24 @@
                 timeout: message.timeout || 0,
                 jar: false,
                 followAllRedirects: true,
-                strictSSL: false,
-                rejectUnauthorized : false
+                rejectUnauthorized : true
             };
 
             if(message.response_timeout != undefined){
                 request_options.response_timeout = message.response_timeout;
             }
+            
+            if(message.validateCertificates != undefined) {
```

Comment:
May be worth ensuring that it is `typeof message.validateCertificates === boolean`. Unless the validation is already set somewhere else and there is a default boolean value somewhere.